engine/runtime/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -386,6 +386,7 @@ object ProgramExecutionSupport {
                   Api.ProfilingInfo.ExecutionTime(e.getNanoTimeElapsed)
                 }.toVector,
                 value.wasCached(),
+                value.isTypeChanged() || value.isFunctionCallChanged(),
```

Comment:
This change + filtering in `SuggestionsHandler` is the gist of this PR. The rest is just noise in tests.

Reply:
Glad that it worked out. One unfortunate thing is that we cannot filter those early in the instrument like it was done originally. But as I understand, we need to propagate them to the language server to handle errors and panics properly.